src/fu-util.c

```diff
@@ -2250,6 +2254,15 @@ main (int argc, char *argv[])
 		return EXIT_FAILURE;
 	}
 
+	/* allow disabling SSL strict mode for broken corporate proxies */
+	if (priv->disable_ssl_strict) {
+		/* TRANSLATORS: try to help */
+		g_print ("%s\n", _("Ignoring SSL strict checks, "
```

Comment:
In my mind it should be scary you're doing things like this.  Should this be `g_print` or `g_printerr`?

Reply:
I think a `g_printerr` with a `WARNING: ` prefix, indeed.